src/schema/me/credit_cards.ts

```diff
@@ -16,10 +16,13 @@ export const CreditCards = {
     { rootValue: { accessToken, meCreditCardsLoader } }
   ) => {
     if (!accessToken) return null
-    return meCreditCardsLoader().then(({ body }) => {
+    const { page, size, offset } = parseRelayOptions(options)
+    const gravityArgs = { page, size, total_count: true }
+
+    return meCreditCardsLoader(gravityArgs).then(({ body, headers }) => {
       return connectionFromArraySlice(body, options, {
-        arrayLength: body && body.length,
-        sliceStart: 0,
+        arrayLength: headers["x-total-count"],
```

Comment:
@mzikherman I updated this to use the header value and correctly calculate offset/etc.